lib/tasks/protected_locations.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+namespace :protected_locations do
+  desc "Sets the protected flag to true for locations with certain location types"
+  task set_default_protected_flag: :environment do
+    protected_locations = ['Building', 'Room', 'Site', 'Car Park', 'Reefer', 'Bin', 'DELIVERED']
```

Comment:
Double check these, these are production environment equivalent of the list of location types that need protecting